src/archanaNJun20/ArrayUnion.java

```diff
@@ -0,0 +1,59 @@
+package archanaNJun20;
+
+public class ArrayUnion {
+	int arraycount=0;
+	double[] getUnionOfArray(double[] array1,double[] array2){
+		int arraycount=0;
```

Comment:
Why you have created two variable with same name? Any specific reason?

Reply:
sorry, my bad.by mistake I declare it twice.